start_cerberus.py

```diff
@@ -100,15 +100,20 @@ def main(cfg):
 
             if slack_integration:
                 weekday = runcommand.invoke("date '+%A'")[:-1]
-                cop_slack_member_ID = config['cop_slack_ID'][weekday]
+                cop_slack_member_ID = config["cerberus"]["cop_slack_ID"][weekday]
                 valid_cops = slackcli.get_channel_members()['members']
 
+                if cop_slack_member_ID in valid_cops:
+                    slack_tag = "<@" + cop_slack_member_ID + ">"
+                elif weekday in ['Saturday', 'Sunday']:
+                    slack_tag = ""
+                else:
+                    slack_tag = "@here "
```

Comment:
Might be overkill to have the `@here`... Maybe we offer the option to include a team alias, or it just throws messages to the channel, similar to the github bot.

Reply:
Sounds good.

Reply:
I have updated it, the slack_team_alias tag is used while reporting failures in the slack channel if the slack_team_alias tag is set in the config else no tag is used. Please have a look @jtaleric @chaitanyaenr